clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/ClinvarMapper.scala

```diff
@@ -245,6 +246,122 @@ class ClinvarMapper(implicit msgCoder: Coder[Msg]) extends Serializable {
     }
   }
 
+  val vaTraitSetMappings = Map(
```

Comment:
See next file for context
```suggestion
  val traitSetMappings = Map(
```